keycloak-themes/README.md

```diff
@@ -54,6 +54,9 @@ e.g. To clear the cache for the moh-app-realm login theme in Keycloak Dev, run t
 rm -rf /data/gfadmin/software/keycloak/domain/servers/server-one/tmp/kc-gzip-cache/m465f/login/moh-app-realm/
 ```
 
+# IDP Outage notification
+In case of a rare event of an IDP outage, the login page can be edited, to inform users about the issue. To display the outage notification, a developer needs to append the word `outage` to the Display Name attribute of the damaged Identity Provider in the corresponding realm. For example, if IDIR IDP is not working, changing Display Name from `IDIR` to `IDIR outage` in the `moh_applications` realm will result in displaying the following message: `Login with IDIR is currently unavailable. IDP team is working on restoring the service.` No server restart is needed. To delete the notification, the Display Name must be set to the previous value.
```

Comment:
```suggestion
In case of a rare event of an IDP outage, the login page can be edited, to inform users about the issue. To display the outage notification, a developer needs to append the word `outage` to the Display Name attribute of the affected Identity Provider in the corresponding realm. For example, if IDIR IDP is not working, changing Display Name from `IDIR` to `IDIR outage` in the `moh_applications` realm will result in displaying the following message: `Login with IDIR is currently unavailable. IDP team is working on restoring the service.` No server restart is needed. To delete the notification, the Display Name must be set to the previous value.
```